### DISCUSSION TITLE
DeDRM plugin doesn't work

### DISCUSSION THREAD

**Discussion by aldval59** [noDRM/DeDRM_tools#252] [General]
Hi everybody,
it's about one month since I have a problem with Calibre DeDRM plugin.
I have bought an ebook from IBS store but when I load it in Calibre the DeDRM plugin doesn't remove DRM so that I have an error message saying that the ebook is protected by DRM and cannot be opened.
I have tried also to load again old ebooks that I had already loaded 3 or 4 month ago with successfully removal of DRM but I had the same error as the latest one.
Anyone has this issue ?
Any suggestion to solve the problem ?
I have 6.10 version of Calibre and 10.0.3 version of DeDRM plugin.
P.S. I can read the ebook using Adobe Digital Edition.
Thanks in advance for the help.

**Comment by ElleKayEm** [comment]
Post a log: https://github.com/noDRM/DeDRM_tools/blob/master/FAQs.md#i-cannot-solve-my-problem-with-the-dedrm-plugin-and-now-i-need-to-post-a-log-how-do-i-do-that
